CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# master
```

Comment:
What do you think of the common pattern
```
- New features
- Bug fixes
- Changes
```

Reply:
I like it, I'll make the change. Where would you put the entries from this PR? In "New Features" or just "Changes"?